packages/firebase/index.d.ts

```diff
@@ -1469,19 +1469,45 @@ declare namespace firebase.appCheck {
   export interface AppCheck {
     /**
      * Activate AppCheck
-     * @param siteKeyOrOrovider - reCAPTCHA sitekey or custom token provider
+     * @param siteKeyOrProvider - reCAPTCHA sitekey or custom token provider
```

Comment:
this is tsdoc format (using `-`), but the doc generator uses jsdoc. I'm not sure if it will parse it correctly. We can try running the generator to verify, or just change to jsdoc format which seems easier.